src/app/models/evt-models.ts

```diff
@@ -105,3 +105,27 @@ export interface NamedEntityRefData extends GenericElementData {
     entityId: string;
     entityType: NamedEntityType;
 }
+
+export interface WitnessesData {
+    witnesses: Map<Witness>;
+    witnessGroups: WitnessGroups;
+}
+
+export interface Witness {
+    id: string;
+    name: Description;
```

Comment:
IMHO you should not use `Description` type in here but directly `GenericElementData[]` or a more specific type. `Description` is intended to be the data model for `<desc>` TEI element and could change in the future if we decide to handle it differently.